### PR TITLE
Cards with S (part 2)

### PR DESCRIPTION
**Pull request by chosta**

### CODE REVIEW

File: src/cards/base/StandardTechnology.ts [hunk 1]
```diff
@@ -20,4 +21,13 @@ export class StandardTechnology implements IProjectCard {
     public play() {
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '156',
+      renderData: CardRenderer.builder((b) => {
+        b.effectBox((eb) => {
+          eb.plate('Standart projects').startEffect.megacredits(3);
```
Comment (kberg) [inline src/cards/base/StandardTechnology.ts:28]: "standart"

File: src/cards/venusNext/Stratopolis.ts [hunk 2]
```diff
@@ -60,4 +65,22 @@ export class Stratopolis implements IActionCard, IProjectCard, IResourceCard {
         },
       );
     }
+
+    public metadata: CardMetadata = {
+      cardNumber: '248',
+      requirements: CardRequirements.builder((b) => b.tag(Tags.SCIENCE, 2)),
+      renderData: CardRenderer.builder((b) => {
+        b.effectBox((eb) => {
+          eb.empty().startAction.floaters(2).secondaryTag(Tags.VENUS);
```
Comment (kberg) [inline src/cards/venusNext/Stratopolis.ts:74]: Does this need an any()?

File: src/cards/venusNext/StratosphericBirds.ts [hunk 3]
```diff
@@ -45,4 +50,21 @@ export class StratosphericBirds implements IActionCard, IProjectCard, IResourceC
       player.addResourceTo(this);
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '249',
+      requirements: CardRequirements.builder((b) => b.venus(12)),
```
Comment (kberg) [inline src/cards/venusNext/StratosphericBirds.ts:55]: Any "special" indicator on this card for the floater?

File: src/cards/venusNext/SulphurExports.ts [hunk 4]
```diff
@@ -30,4 +32,13 @@ export class SulphurExports implements IProjectCard {
...
       return undefined;
     }
+
+    public metadata: CardMetadata = {
+      cardNumber: '250',
+      renderData: CardRenderer.builder((b) => {
+        b.venus(1).br;
+        b.productionBox((pb) => pb.megacredits(1).slash().venus(1).played);
+      }),
+      description: 'Increase Venus 1 step. Increase your MC production1 step for each Venus tag you have, including this.',
```
Comment (kberg) [inline src/cards/venusNext/SulphurExports.ts:42]: "production1"

**Review by kberg — APPROVED** [review]

File: src/cards/base/SpaceMirrors.ts [hunk 5]
```diff
@@ -25,4 +28,14 @@ export class SpaceMirrors implements IActionCard, IProjectCard {
       player.addProduction(Resources.ENERGY);
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '076',
+      requirements: CardRequirements.builder((b) => b.tag(Tags.SCIENCE, 5)),
```
[the remark this answers is not in the document]
Comment (chosta) [reply src/cards/base/SpaceMirrors.ts:33]: There are actually no requirements for this card.

File: src/cards/venusNext/Stratopolis.ts (hunk 2 above)
Comment (chosta) [reply src/cards/venusNext/Stratopolis.ts:74]: Good point but this needs to be a separate issue. I think the original card design does not have the red border. Will ask in discord.

File: src/cards/venusNext/StratosphericBirds.ts (hunk 3 above)
Comment (chosta) [reply src/cards/venusNext/StratosphericBirds.ts:55]: Again not seen in the original design. Will ask.

**Pull request merged**